README.md

```diff
@@ -109,6 +118,56 @@ var client = contentful.createClient({
 // ....
 ```
 
+## Webpack bundle size optimization
+
+We are constantly working on smaller bundle sizes for our customers to allow them to build faster and less bandwidth consuming apps. Since version 4 the contentful-management SDK is exported as modules version, also. This version is automatically picked by webpack to enable tree-shaking.
+
+For further optimization of your app size, we recommend the following:
+
+### Step 1: Transform generic lodash requires to cherry-picked ones:
+
+This can be achieved via [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) and can already help to reduce the bundle size.
+
+### Step 2: Remove unnecessary lodash functionality:
```

Comment:
`common knowledge` - I doubt that. Configuring webpack is still magic for most of the web developers. Sharing the knowledge with less-webpack-skilled is IMHO a good service since it helps to squeeze the most out of our libs.